doc/team_practices.md

```diff
@@ -26,54 +74,94 @@ In the workflow below, there are multiple roles to be played:
-Otherwise, fork the 18F repo. The branch naming convention is *storyId*-*short-description*.
-Example: `git checkout -b 123456-fix-timezones`
+When a developer starts a story, they move the card to this list and
+self-assign. Each developer should aim to have no more than 2-3 stories in
+process at a time.
 
-1. Write your code and tests and documentation. You can
-[read about running tests](https://github.com/18F/C2/blob/master/doc/setup.md#running-tests)
-and [view the doc folder](https://github.com/18F/C2/tree/master/doc).
+The author writes code and tests and documentation. Read [read about running
+tests](https://github.com/18F/C2/blob/master/doc/setup.md#running-tests) and
+[view the doc folder](https://github.com/18F/C2/tree/master/doc).
 
-1. Push your branch to Github.
+**`Code Review`**
 
-1. Create a Pull Request. The PR should reference the story URL and include a brief description
-of the changes, including any rationale for how/why the story is addressed in the way it is. Use the format
-`[#STORYID]` or `[Delivers #STORYID]` or `[Fixes #STORYID]` in the PR title.
-See the [story tracker
-documentation](https://www.pivotaltracker.com/help/api?version=v5#Tracker_Updates_in_SCM_Post_Commit_Hooks).
-Consider adding any notes relevant to testing the change, especially for live QA
+When a pull request is opened for a change, the author moves the Trello card to
+the code review list. Once a card is moved to this list, the team uses
+assignment in the GitHub pull request rather than in Trello.
+
+The PR should link to the Trello card and include a brief description of the
+changes, including any rationale for how/why the story is addressed in the way
+it is.
+
+The author should add any notes relevant to testing the change, especially for live QA
 testing.
 
-1. Click **Finish** on the story in the story tracker.
+The author should link to the pull request in a comment on the Trello card.
+
+Someone from the team should review the PR. If you do not get feedback within
+24 hours, assign the PR to a team member for review. The Reviewer should
+consider the automated testing status and Code Climate reports, in addition to
+checking the code for architectural consistency, style and legibility. Code
+reviews are encouraged early in the development process; you can always create
+a PR with a `[WIP]` prefix before you are ready to deliver, and ask for review
+of work-in-progress. The Reviewer is encouraged to indicate a "Ship it" (or
+your favorite Ship It emoticon) Consider pointing out the awesomeness of the
+code, too.
+
+If the committer paired with someone on the story, the teammate can certainly
+give a Ship It. That said, the pair is welcome to solicit additional PR
+feedback from the rest of the team. If verbal approval is given by a teammate,
+the committer may comment `@TEAMMATE ship it` before merging the PR.
+
+If a card is code reviewed and requires more work before it is ready for QA,
```

Comment:
in this context, I was referring to the next step in the process, which is the Trello list called "QA" -- what about changing to

> If a card is code reviewed and requires more work before it is ready for the QA step,